src/components/IconRounded/WIconRounded.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div
+    class="flex items-center justify-center border border-black-950 rounded-full w-full p-md bg-wit-blue-500 hover:bg-wit-blue-300 transition-colors"
+    :class="customWidth"
+    :style="customStyles"
+  >
+    <slot></slot>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue'
+
+const props = defineProps({
+  width: {
```

Comment:
As we are going to set height too, maybe it is best to change prop name?
```suggestion
  iconSize: {
```